paris/src/main/java/com/airbnb/paris/proxies/ViewProxy.java

```diff
@@ -214,6 +223,33 @@ public void setBackground(@Nullable Drawable drawable) {
         getView().setBackground(drawable);
     }
 
+    @Attr(R2.styleable.Paris_View_android_backgroundTint)
+    public void setBackgroundTint(ColorStateList colorStateList) {
+        ViewCompat.setBackgroundTintList(
+                getView(),
+                colorStateList
+        );
+    }
+
+    @Attr(R2.styleable.Paris_View_android_backgroundTintMode)
+    public void setBackgroundTintMode(int tintMode) {
+        ViewCompat.setBackgroundTintMode(getView(), parseTintMode(tintMode));
```

Comment:
What happens if the tint mode is unsupported/null? Can you add a test for it?

Reply:
if the tint mode is null, then the tint mode is cleared.